website/docs/advanced/api_access.md

```diff
@@ -11,7 +11,9 @@ Please refer to [Create token](../user_guide/api-token) on how to create an API
 
 Please note that it may take up to 60 seconds for the new key to propagate to all Unleash instances due to eager caching.
 
-> If you need an API token to use in a client SDK you should create a "client token" as these have less access.
+:::note
+If you need an API token to use in a client SDK you should create a "client token" as these have fewer access rights.
+:::
```

Comment:
I was not aware of the `:::note` syntax, nice!

Reply:
Yeah, docusaurus has some really nice features built in! 

For admonitions, you also have tip, info, caution, and danger: https://docusaurus.io/docs/markdown-features/admonitions